src/ng/parse.js

```diff
@@ -401,7 +400,7 @@ AST.prototype = {
 
   program: function() {
     var body = [];
-    while (true) {
+    for (;;) {
```

Comment:
This is a bit unusual (at least to me) and could be explained with a comment. Or the no-constant-condition rule could be disabled for this line (while keeping while(true)